### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by hluijk** [johngodley/redirection#657]
```
Plugin: 3.0.1
WordPress: 4.9.2
PHP: 5.4.16
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36

Error: Cookie nonce is invalid
Action: redirection/v1/setting GET
Code: 403 
Raw: {"code":"rest_cookie_invalid_nonce","message":"Cookie nonce is invalid","data":{"status":403}}

Error: Cookie nonce is invalid
Action: redirection/v1/plugin GET
Code: 403 
Raw: {"code":"rest_cookie_invalid_nonce","message":"Cookie nonce is invalid","data":{"status":403}}
```

**Comment by johngodley** [comment]
Can you try the 3.1 beta?

https://github.com/johngodley/redirection/releases/tag/3.1

**Comment by mercutiom** [comment]
I'm getting the exact same error after the recent update. Installing the beta version didn't work to fix it. It gave me this error:

Something went wrong 🙁
Your server returned a 403 Forbidden error which may indicate the request was blocked. Are you using a firewall or a security plugin?

Please take a look at the plugin status. It may be able to identify and "magic fix" the problem.
Redirection is unable to talk to your REST API. If you have disabled it then you will need to enable it.
Security software may be blocking Redirection. You will need to configure this to allow REST API requests.
Caching software, in particular Cloudflare, can cache the wrong thing. Try clearing all your caches.
Please temporarily disable other plugins! This fixes so many problems.
None of the suggestions helped
If this is a new problem then please either create a new issue or send it in an email. Include a description of what you were trying to do and the important details listed below. Please include a screenshot.

I tried those fixes to no avail. Suggestions?

**Comment by johngodley** [comment]
I've now released version 3.1. I will close this ticket for now, but please reopen if the problem persists

@mercutiom if it doesn't fix your problem can you open a seperate ticket?
